recipes/glimpse-bio/build.sh

```diff
@@ -1,20 +1,29 @@
 #!/bin/bash
 
+COMMIT_VERS="b3ff92f"
+COMMIT_DATE="2022-12-07"
+__COMMIT_ID__="$COMMIT_VERS"
+__COMMIT_DATE__="$COMMIT_DATE"
+
+for subdir in chunk split_reference phase ligate
 
-for subdir in chunk concordance ligate phase sample snparray
 do
     pushd $subdir
     # Build the binaries
     make \
         -j 4 \
-        DYN_LIBS="-lz -lpthread -lbz2 -llzma -lcurl -lhts -ldeflate" \
+        DYN_LIBS="-lz -lpthread -lbz2 -llzma -lcurl -lhts -ldeflate -lm" \
         CXX="$CXX -std=c++11" \
         CXXFLAG="$CXXFLAGS ${PREFIX}" \
         LDFLAG="$LDFLAGS" \
         HTSLIB_INC="$PREFIX" \
         HTSLIB_LIB="-lhts" \
+        BOOST_INC="/usr/include"\
         BOOST_LIB_IO="-lboost_iostreams" \
         BOOST_LIB_PO="-lboost_program_options" \
+        BOOST_LIB_SE="-lboost_serialization" \
+        __COMMIT_ID__="b3ff92f" \
+        __COMMIT_DATE__="2022-12-07" \
```

Comment:
Use the variables used above.
```suggestion
        __COMMIT_ID__="${__COMMIT_ID__}" \
        __COMMIT_DATE__="${__COMMIT_DATE__}" \
```